notification-stream/src/types/index.ts

```diff
@@ -35,11 +35,25 @@ export interface IUserExternalId {
 export interface IPayload {
     url: string;
     image_url: string;
+    topic_id?: string;
+    description?: string;
+    title?: string;
     content: IPayloadContent;
     target_content_uuid: string;
 }
 
 export interface IPayloadContent {
     uuid: string;
-    body: string;
+    body?: string;
```

Comment:
Are there scenarios where there is no body?

Reply:
it's optional in the spec. but yeah sounds weird.